src/store/scan/scan.effects.ts

```diff
@@ -196,6 +208,138 @@ const goToPayPro =
     }
   };
 
+const handleUnlock =
+  (data: string): Effect =>
+  async dispatch => {
+    const invoiceId = data.split('i/')[1].split('?')[0];
+    const network = data.includes('test') ? Network.testnet : Network.mainnet;
+    const result = await dispatch(unlockInvoice(invoiceId, network));
+
+    if (result === 'unlockSuccess') {
+      dispatch(goToPayPro(data));
+      return;
+    }
+
+    const {host} = new URL(GetPayProUrl(data));
+    try {
+      const invoice = await axios.get(
+        `https://${host}/invoiceData/${invoiceId}`,
+      );
+      if (invoice) {
+        dispatch(goToPayPro(data));
+        return;
+      }
+    } catch {}
+    switch (result) {
+      case 'pairingRequired':
+        navigationRef.navigate('Auth', {
+          screen: 'Login',
+          params: {
+            onLoginSuccess: () => {
+              const parentNav = navigationRef.getParent();
+
+              if (parentNav?.canGoBack()) {
+                parentNav.goBack();
+              } else {
+                navigationRef.navigate('Tabs', {screen: 'Home'});
+              }
+
+              dispatch(incomingData(data));
+            },
+          },
+        });
+        break;
+
+      // needs verification - send to bitpay id verify
+      case 'userShopperNotFound':
+      case 'tierNotMet':
+        dispatch(
+          showBottomNotificationModal({
+            type: 'warning',
+            title: t('Connect Your BitPay ID'),
+            enableBackdropDismiss: false,
+            message: t(
+              'To complete this payment, please login with your BitPay ID.',
```

Comment:
Did these translations make it into the translation request? If not we need to queue them up for the next translations batch

Reply:
no not to the current one. yes, there is one more PR which add onGoingProcess translations, this will get into that boat.